index.html

```diff
@@ -1256,8 +1256,96 @@
       <section>
         <h3>Usage Patterns</h3>
 
-        <p class="issue">
-When verifiable claims are used can be used for correlation purposes.
+        <p>
+Despite the best efforts to assure privacy, the actual use of verifiable
+claims can potentially lead to de-anonymization and a loss of privacy.
+This correlation can occur:
+        </p>
+
+        <ol>
+          <li>
+When the same claim is presented to the same inspector more than once –
+that inspector could infer that the holder is the same individual.
+          </li>
+          <li>
+When the same claim is presented to different inspectors, and either those
+inspectors collude or a third party has access to transaction records from
+both inspectors – the observant party could infer that the individual
+presenting the claims is the same person at both services, i.e., the
+accounts are controlled by the same person.
+          </li>
+          <li>
+When the same subject ID of a claim refers to the same subject across
```

Comment:
Agreed. This was the tension I initially struggled with, seeing the need to frame the bigger picture: intended and unintended correlation alongside means for preventing correlation (best practices and limitations thereof).  If we added that section, we could update the rest for better, simpler flow.